controllers/storageautoscaler/scraper_test.go

```diff
@@ -156,6 +164,27 @@ func TestUpdateSyncMap(t *testing.T) {
 
 		// check if the sync map is updated
 		usage, _ := syncMap.Load("ssd")
-		assert.Equal(t, 0.70, usage)
+		assert.Equal(t, 0.70, usage.(OsdUsage).HighestUsage)
+	})
+}
+
+func TestTotalUsageGreaterThanAdjustedThreshold(t *testing.T) {
+	t.Run("test total usage greater than threshold minus 10 percent", func(t *testing.T) {
+		deviceClassUsage := OsdUsage{
+			TotalUsage:   1.80,
+			TotalOsd:     3,
+			HighestUsage: 0.60,
+		}
+		result := totalUsageGreaterThanAdjustedThreshold(deviceClassUsage, 50)
+		assert.True(t, result)
+	})
+	t.Run("after resize test total usage less than threshold minus 10 percent", func(t *testing.T) {
+		deviceClassUsage := OsdUsage{
+			TotalUsage:   1.90,
+			TotalOsd:     6,
+			HighestUsage: 0.55,
+		}
```

Comment:
How about adding a test case that is barely over 40% adjusted threshold? Boundary checking is helpful for computations like this.